src/contracts/README.md

```diff
@@ -0,0 +1,13 @@
+## Contracts for RIP-7560 and ERC-4337
+
+### Compile
+
+```shell
+$ forge compile
+```
+
+### Deploy
+
+```shell
+$ forge script script/Deploy.s.sol:CounterScript --rpc-url $RPC_URL --private-key $PRIVATE_KEY --broadcast
```

Comment:
I don't see CounterScript.. Shouldn't it be like this?
forge script script/Deploy.s.sol:DeployScript --rpc-url $RPC_URL --private-key $PRIVATE_KEY --broadcast